code/game/objects/items.dm

```diff
@@ -244,8 +244,9 @@
 
 /obj/item/throw_impact(atom/hit_atom)
 	..()
-	if(drop_sound)
-		playsound(src, drop_sound, 50, 0)
+	for(var/mob/M in player_list)
+		if(drop_sound && (M.client.prefs.asfx_togs & ASFX_DROP))

```

Comment:
`M.client.prefs.asfx_togs & ASFX_DROP` should be made into a macro or a function.

Reply:
I... don't know what that means

Reply:
I would make it into function
```
proc/check_drop_pref(mob/M)
    return (M.client.prefs.asfx_togs & ASFX_DROP)
```